docs/quickstart/bootstrap.rst

```diff
@@ -36,6 +36,9 @@ Configuration
       networks:
         controlPlane: <CIDR-notation>
         workloadPlane: <CIDR-notation>
+      proxies:
```

Comment:
Since we only use these values to *specifically* configure `containerd`, not any other services that may be running, or containers, or whatnot, maybe using a more specific name would be wise? WDYT @scality/metalk8s

Reply:
Yes maybe make sense to know just by reading the file that this proxy will only be used by the container engine to fetch images. 
Maybe something like
- `registry_proxies`
- `extra_container_images_proxies`
- `images_proxies`

(I'm not sure we want to talk about containerd here)

Reply:
Well, no, I don't think we need to do *multiple* proxy declarations (if there's some proxy in place, there is one, used for all traffic).
However, as-is, it may not be clear we *only* use this configuration *for one very specific service*.

Reply:
Oh no not saying we need multiple just suggest few key name

Reply:
I've put `proxies` to be more general, because I don't know if we'll use this entry to set the proxy on other stuff in the future.
But ok, I will change it.

Reply:
That makes sense, and maybe `proxies` *is* the right key. But in that case, it should be explicitly documented where exactly this configuration is used/applied, such that there can be no false assumptions about other subsystems being configured using it.

Reply:
It's already the case just bellow in the doc no ?